README.md

```diff
@@ -45,11 +49,12 @@ Here are the instructions to build SciLexer.dll (for both 32-bit & 64-bit) for N
 
  1. Download the [Boost source code](http://sourceforge.net/projects/boost/files/boost/1.55.0/).
     v1.55 should be used with VS 2013. Then unzip it. In my case, `boost_1_55_0` is copied in `C:\sources\`
- 2. Go to `scintilla\boostregex\` then run BuildBoost.bat with your boost path.
+ 2. You may need to follow this step- Open command prompt, run vcvarsall.bat (e.g. "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat").
+ 3. From *same* command prompt, Go to `scintilla\boostregex\` then run BuildBoost.bat with your boost path.
     In my case: `BuildBoost.bat C:\sources\boost_1_55_0`
 	If you are compiling a 64 bit Scintilla under your *VS2013 x64 Native tool command prompt*, add `-x64` flag.
 	In my case: `BuildBoost.bat C:\sources\boost_1_55_0 -x64`
- 3. Go in `scintilla\win32\` then run `nmake -f scintilla.mak`
+ 4. Go in `scintilla\win32\` then run `nmake -f scintilla.mak`
 

```

Comment:
Yes, I can understand that you'd want to extend the instructions in the section "To build `SciLexer.dll` _with_ boost". But why not extend them as done in the next section "To build `SciLexer.dll` _without_ boost"? Only the applicable ones, to be clear.

FYI  you don't need to run `vcvarsall.bat` to be able to build Boost. You need it to be able to build Scintilla (i.e. `SciLexer.dll`) though.